_rules/SC4-1-2-iframe-unique-name.md

```diff
@@ -0,0 +1,206 @@
+</iframe>
+<iframe title="List of Contributors" src="../test-assets/SC4-1-2-iframe-unique-name-doc2.html">
+</iframe>
+```
+
+
+#### Pass example 5
+
+`frames` are allowed to have the same `title` across different document contexts.
+
+```html
+<iframe title="List of Contributors" src="../test-assets/SC4-1-2-iframe-unique-name-doc1.html">
+</iframe>
+<iframe title="List of Contributors 2" src="../test-assets/SC4-1-2-iframe-unique-name-doc2.html">
+  <iframe title="List of Contributors" src="../test-assets/SC4-1-2-iframe-unique-name-doc1.html">
+  </iframe>
+</iframe>
+```
+
+
+### Failed
+
+#### Fail example 1
+
+Multiple frames having the same `title` within a given document context.
+
+```html
+<iframe title="List of Contributors" src="../test-assets/SC4-1-2-iframe-unique-name-doc1.html">
+</iframe>
+<iframe title="List of Contributors" src="../test-assets/SC4-1-2-iframe-unique-name-doc2.html">
+</iframe>
+```
+
+#### Fail example 2
+
+Multiple frames having the same `aria-label` within a given document context.
+
+```html
+<iframe aria-label="List of Contributors" src="../test-assets/SC4-1-2-iframe-unique-name-doc1.html">
+</iframe>
+<iframe aria-label="List of Contributors" src="../test-assets/SC4-1-2-iframe-unique-name-doc2.html">
+</iframe>
+```
+
+#### Fail example 3
+
+Multiple frames having the same name given by `title` and `aria-label` within a given document context
+
+```html
+<iframe title="List of Contributors" src="../test-assets/SC4-1-2-iframe-unique-name-doc1.html">
+</iframe>
+<iframe aria-label="List of Contributors" src="../test-assets/SC4-1-2-iframe-unique-name-doc2.html">
+</iframe>
+```
+
+### Inapplicable
+
+#### Inapplicable example 1
+
+Accessible name provided is empty.
+
+```html
+<iframe title="" src="../test-assets/SC4-1-2-iframe-unique-name-doc1.html">
+</iframe>
+<iframe aria-label="" src="../test-assets/SC4-1-2-iframe-unique-name-doc3.html">
+</iframe>
+```
+
+#### Inapplicable example 2
+
+Usage of `alt` attribute to describe content is not valid.
+
+```html
+<iframe alt="Some" src="../test-assets/SC4-1-2-iframe-unique-name-doc1.html">
+</iframe>
+```
+
+#### Inapplicable example 3
+
+Does not apply to non `iframe` elements.
+
+```html
+<object title="List of Contributors" src="../test-assets/SC4-1-2-iframe-unique-name-doc1.html">
+</object>
+<object aria-label="List of Contributors Clone" src="../test-assets/SC4-1-2-iframe-unique-name-doc3.html">
+</object>
+```
+
+#### Inapplicable example 4
+
+No accessible name is provided
+
+```html
+<iframe src="../test-assets/SC4-1-2-iframe-unique-name-doc1.html">
+</iframe>
+<iframe src="../test-assets/SC4-1-2-iframe-unique-name-doc3.html">
+</iframe>
+```
+
+#### Inapplicable example 5
+
+Does not apply to `iframe` elements that are not exposed to assistive technologies.
+
+```html
+<iframe aria-hidden="true" title="Document One" src="../test-assets/SC4-1-2-iframe-unique-name-doc1.html">
```

Comment:
This frame is still exposed to assistive technologies. Frames are focusable elements.